osidb/models.py

```diff
@@ -1753,6 +1764,25 @@ def create_flawcomment(flaw, external_system_id, comment, **extra_fields):
             flawcomment.meta_attr = comment
             return flawcomment
         except ObjectDoesNotExist:
+            # The "text" field is required for full validation, so save it if the caller of
+            # this method forgets to explicitly copy it over from meta_attr.
+            extra_fields["text"] = extra_fields.get("text") or comment.get("text")
+
+            # The "order" field is required for full validation, but bugzilla calls it "count".
+            extra_fields["order"] = extra_fields.get("order") or comment.get("count")
+
```

Comment:
@osoukup 's comment at https://github.com/RedHatProductSecurity/osidb/pull/209#discussion_r1209402917 is still valid, it is expected of callers of this method to provide all the necessary fields explicitly in extra_fields, and in fact we do so in bzimport https://github.com/RedHatProductSecurity/osidb/blob/f41cf220cbdf0cb197dc404b9fef48996e5d0393/collectors/bzimport/convertors.py#L702-L719

Reply:
I'll fix the invalid call in `test_flaw.py` and remove these copy operations then.

Reply:
Resolved in commit a3415f9b5cde43c7e4fb3ac8476e8791fd710365